examples/planning/generate_keyword_ideas.py

```diff
@@ -0,0 +1,168 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example generates keyword ideas from a list of seed keywords."""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+# Location IDs are listed here: https://developers.google.com/adwords/api/docs/appendix/geotargeting
+# and they can also be retrieved using the GeoTargetConstantService as shown
+# here: https://developers.google.com/google-ads/api/docs/targeting/location-targeting
+_DEFAULT_LOCATION_IDS = '1023191'  # location ID for New York, NY
```

Comment:
since the argument in the call is in plural form (_IDS), how would you specify more than one ID? ['1023191', '123']?

Reply:
Since this is a default value for the command line input it would need to be a comma-separated list of strings.

Reply:
it's a nit. If the variable name is plural, we should show more than one value. consider it for future reference.